tools/build_python_cffi.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Running `./tools/build_python_cffi.sh` on my end, I spotted 2 differences:
```
$ ./tools/build_python_cffi.sh | grep wally_tx_get_total_output_satoshi
    ('wally_tx_get_total_output_satoshi', c_int, [POINTER(wally_tx), c_ulonglong_p]),
$ cat src/test/util.py | grep wally_tx_get_total_output_satoshi
    ('wally_tx_get_total_output_satoshi', c_int, [POINTER(wally_tx), c_ulonglong]),
```
```
$ ./tools/build_python_cffi.sh | grep wally_unknowns_map_find
    ('wally_unknowns_map_find', c_int, [POINTER(wally_unknowns_map), c_void_p, c_ulong, c_ulong_p]),
$ leonardo@leonardo-TECRA-A50-C:~/greenaddress/libwally-core$ cat src/test/util.py | grep wally_unknowns_map_find
```

Reply:
Ive updated the wrapper generator again and re-generated if you could review again, thanks!